### ISSUE TITLE
L1 penalty for keras linear_reg

### ISSUE THREAD

**Issue by jaredlander** [tidymodels/parsnip#198]
The documentation states

> For keras models, this corresponds to purely L2 regularization (aka weight decay) ...

And the code does indeed show that only `keras::regularizer_l2() is being used.

```
model %>% keras::layer_dense(units = hidden_units, activation = act, 
            input_shape = ncol(x), kernel_regularizer = keras::regularizer_l2(decay), 
            kernel_initializer = keras::initializer_glorot_uniform(seed = seeds[1]))
```

However, `{keras}` has the `regularizer_l1_l2()` function which allows both L1 and L2 penalties.

Can that be used instead of `regularizer_l2()` to allow either lasso or ridge or both?
